app/src/main/java/com/codesoom/assignment/HttpStatus.java

```diff
@@ -0,0 +1,24 @@
+package com.codesoom.assignment;
+
+/**
+ * Http 상태코드를 정의합니다.
+ *
+ * @see <a href="https://datatracker.ietf.org/doc/html/rfc2616">RFC 2616</a>
+ */
+public enum HttpStatus {
+    OK(200),
+    CREATED(201),
+    NO_CONTENT(204),
+    NOT_FOUND(404),
+    INTERNAL_SERVER_ERROR(500);
+
+    private final int code;
+
+    HttpStatus(int code) {
+        this.code = code;
+    }
+
+    public int code() {
```

Comment:
보통은 이렇게 하지만 반드시 지켜야 할 필요는 없습니다. (저는 쓸데없는 컨벤션이라 생각합니다)
```suggestion
    public int getCode() {
```